op-deployer/pkg/deployer/pipeline/seal_l1_dev_genesis.go

```diff
@@ -48,6 +48,9 @@ func SealL1DevGenesis(env *Env, intent *state.Intent, st *state.State) error {
 		},
 		L1ChainID:          eth.ChainIDFromUInt64(intent.L1ChainID),
 		L1PragueTimeOffset: l1DevParams.PragueTimeOffset,
+		L1OsakaTimeOffset:  l1DevParams.OsakaTimeOffset,
+		L1BPO1TimeOffset:   l1DevParams.BPO1TimeOffset,
```

Comment:
Should we put L1BPO2TimeOffset here, since testnets (and our betanet) will activate it?

Reply:
I've only added the forks that the tests rely on. They won't read the `BPO2TimeOffset`, for example.